core/server/apps/default-cards/cards/markdown.js

```diff
@@ -1,16 +1,44 @@
 var SimpleDom   = require('simple-dom'),
     tokenizer   = require('simple-html-tokenizer').tokenize,
+    jsdom       = require('jsdom').jsdom,
```

Comment:
Could you please move all third party dependencies into the render function?
The background is that we don't have to load them on bootstrap.
This slows down the bootstrap time by ~200ms. Thanks 👍 

Furthermore, the global variables e.g. html, doc can be local variables.